source/operations/checklists/software.rst

```diff
@@ -102,4 +102,8 @@ Post Install Tasks
 
    * - :octicon:`circle`
      - | Authenticate to MinIO with :ref:`Security Token Service (STS) <minio-security-token-service>`
-       | Enabling this requires MinIO support.
\ No newline at end of file
+       | Enabling this requires MinIO support.
+
+.. include:: /includes/common-admonitions.rst
```

Comment:
Maybe at the end of post-install?

Reply:
Sure, I wasn't really sure where to put it on this particular page.